example_problems/nonrelativistic_boltzmann/linear_modes_test/perturbed_temperature_tests/1D1V/landau_damping/initialize.py

```diff
@@ -0,0 +1,79 @@
+"""
```

Comment:
Please put a README inside each test folder describing the test in detail. I'm not sure what `perturbed_temperature_tests` are, and then why is `landau_damping` a subset of this? Are you using `landau_damping` to indicate zero collisions and non-zero fields? If so, change to something like `zero_collisions_with_fields`

Reply:
Actually, I'm thinking I'll delete this folder. Just want the `linear_modes_tests` to have `perturbed_density_sinusoidal`, and `perturbed_moments_gaussian`. Think that covers it. But I'll add a README to those folders